lib/compute/zone.js

```diff
@@ -483,6 +494,16 @@ Zone.prototype.createInstanceGroup = function(name, options, callback) {
  * };
  *
  * //-
+ * // Create a new instance using the latest Debian version from your project as the source image
+ * // for a new boot disk.
+ * //-
+ * var config = {
+ *   os: 'your-project-id-or-name/debian',
+ *   http: true,
+ *   tags: ['debian-server']
+ * };
+ *
+ * //-
```

Comment:
I think we should put this example first, so that the next section, where it says "The above object will auto-expand..." will be accurate.